clients/admin-ui/src/features/common/custom-reports/CustomReportCreationModal.tsx

```diff
@@ -26,7 +26,7 @@ interface CustomReportCreationModalProps {
   isOpen: boolean;
   handleClose: () => void;
   tableStateToSave: CustomReportTableState | undefined;
-  columnMapToSave: Record<string, string> | undefined;
+  columnMapToSave: Record<string, Record<string, any>> | undefined;
```

Comment:
We want to try to avoid `any` whenever possible. I would suggest adding a new interface to the `types` file. Something like:
```
export interface ColumnVisibility {
  label?: string;
  enabled?: boolean;
}
```
and then replacing all of these `any` instances with `ColumnVisibility`
```suggestion
  columnMapToSave: Record<string, Record<string, ColumnVisibility>> | undefined;
```